### PR TITLE
Fix #1843 - Improve phrasing in metadata

### LINKED ISSUE #1843: We need to review the text for all impact function metadata.py files
# Problem

Many of the phrases use incorrect grammar - we want our release to be filled with beautifully constructed sentances.

# Proposed solution

@timlinux to review all IF metadata files and make a PR with improved writing. @Charlotte-Morgan feel free to give my changes a once over when they are in.

### PR DESCRIPTION
**Pull request by timlinux**
Please note that this is not an exhaustive overhaul - we need to make a consolidated effort to improve the metadata. This also fixes somewhat related #1843 issue where we have production IF called 'experimental'. Also fix for dock layout heading being obscured in OSX (regression of #1161)

### CODE REVIEW

**Comment by timlinux** [comment]
@cchristelis please review

File: safe/impact_functions/inundation/flood_population_evacuation_raster_hazard/impact_function.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def _tabulate(self, counts, evacuated, minimum_needs, question,
             TableRow(tr('* Number is rounded up to the nearest %s') % (
                 rounding_evacuated)),
             TableRow(tr('Map shows the numbers of people needing evacuation')),
-            TableRow(tr('Table below shows the weekly minimum needs for all '
+            TableRow(tr('Table shows the weekly minimum needs for all '
                         'evacuated people'))]
```
Comment (cchristelis) [inline safe/impact_functions/inundation/flood_population_evacuation_raster_hazard/impact_function.py]: Pull left. #Pylintpolice.

**Comment by timlinux** [comment]
test this please, jenkins

**Comment by akbargumbira** [comment]
test this please, jenkins

**Pull request merged**